src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,122 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEAFULT_SIZE = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    int size;
+
+    private Entry<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Entry[DEAFULT_SIZE];
+        size = 0;
+    }
+
+    private static class Entry<K, V> {
+        K key;
+        V value;
+        Entry<K, V> next;
+
+        Entry(K key, V value, Entry<K, V> next) {
```

Comment:
The constructor of 'Entry' does not need to be public since it is a private inner class and is only used within 'MyHashMap'. Consider changing the access modifier to 'private'.